src/apiProxy/apiConfigCreators.js

```diff
@@ -71,6 +71,16 @@ export function event(params) {
 }
 
 /**
+ * admin endpoints
+ */
+export function admin(params) {
+	return {
+		endpoint,
```

Comment:
yeah, probably not using the latest meetup-web-platform - `v0.10.436`.

When you upgrade, you might get a `action$... ofType is not a function` or something like that. You'll need to `import rxjs;` at the top of your `browser-app-entry` and `server-app-entry` files to fix that, but I think the rest of the upgrade should be straightforward [more info here](https://github.com/meetup/meetup-web-platform/blob/master/CHANGELOG.md)